docs/03-endpoints/api-admin/permissions.md

```diff
@@ -222,8 +223,53 @@ Therefore, it is not possible to create new default object access permissions
 for the ProjectAdmin and ProjectMember groups of a project. 
 However, the default permissions set for these groups can be modified; see below for more information.
 
+### Updating an existing Default Object Access Permission
+
+- `PUT: /admin/permissions/doap/<doap_permissionIri>` to change the attributes of an existing default object
+  access permission, identified by its IRI `<doap_permissionIri>`.
+
+This is an example of a request body to update an existing default object access permission:
+
+```json
+{
+  "forProperty" : "http://api.dasch.swiss/ontology/00FF/images/v2#hasTitle",
+  "forResourceClass": "http://api.dasch.swiss/ontology/0803/incunabula/v2#Book",
+  "forGroup": null, 
+  "hasPermissions": [
+    {
+      "additionalInformation": "http://www.knora.org/ontology/knora-admin#ProjectMember",
+      "name": "D",
+      "permissionCode": 7
+    }
+  ]
+}
+```
+
+All attributes of the default object access permission are optional and may be combined.
+
+!!! note
+    Only certain combinations of attributes are allowed. Only exactly one of the following combinations is allowed:
+
+      - `forGroup`  <!-- markdownlint-disable MD046 -->
+      - `forResourceClass`
+      - `forProperty`
+      - `forResourceClass` and `forProperty`
+
+If the combination of attributes is not allowed, the request will fail with a `400 Bad Request` error.
+Any valid combination of attributes will replace the existing values.
+
+If present, the `hasPermissions` property must contain all permission types that must be granted. See [a complete description of object access
+permission types](../../05-internals/design/api-admin/administration.md#default-object-access-permissions).
+This is also described in the [Creating New Default Object Access Permissions](#creating-new-default-object-access-permissions) section.
+
+The response is the updated default object access permission with its new attributes and is the same as when
+[creating a new default object access permission](#creating-new-default-object-access-permissions).
+
 ### Updating a Permission's Group
 
+!!! note
```

Comment:
To put more emphasis on it, I would perhaps choose a warning (https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:warning)

```suggestion
!!! warning
```